src/mod/server.mod/server.c

```diff
@@ -2156,6 +2146,19 @@ char *server_start(Function *global_funcs)
     module_undepend(MODULE_NAME);
     return "This module requires Eggdrop 1.8.0 or later.";
   }
+/* Some pre-compiled OpenSSL libs (SunOS, I'm looking at you) omit ECC keys
+   due to export restrictions. This prevents us from being able to use the
+   NIST256P method */
+#ifdef TLS
+#ifndef HAVE_EVP_PKEY_GET1_EC_KEY
+if (sasl) {
```

Comment:
what about:

```
.set sasl
Currently: 0
.set sasl 1
Ok, set.
```

? I think this block of code has to happen for Tcl_TraceVar / TCL_TRACE_WRITES on variable sasl instead.

Reply:
We had talked about that earlier- this is why we are leaving the error messages at run-time as well. I think that's sufficient to cover what I consider to be a corner-case.